README.md

```diff
@@ -1 +1,48 @@
-# Data Science Stack
+<div align="center">
+
+# Data Science Stack ✨
+
+*Making it seamless to run GPU enabled containerized ML Environments*
+
+</div>
+
+## Overview
+
+The Data Science Stack (DSS) aims to make it seamless for everyone to jump into an ML Environment within minutes and be able to utilise their GPUs.
+
+DSS is a ready-made environment which allows everyone to run ML workloads on the laptop.
+It gives easy access to a solution to develop and optimise ML models, that leverages the GPUs of the laptop by
+enabling users to utilise different ML environment images based on their needs.
+
+The DSS is a stack that includes 
+- a container orchestration system ([microK8s](https://microk8s.io/) snap)
+- out-of-the box containerized ML Environments
+- an intuitive CLI which streamlines the management of those containers (data-science-stack snap)
+
+The container orchestration system also handles how to integrate with the host's
+GPU and drivers, so the containerized environments can only focus on user-space libraries.
+
+## Features
+
+- Containerized environment management
+- Seamless GPU utilization
+- Out-of-the box ML Environments
+- Easy data passing between local machine and containerized ML Environments
+- [MLFlow](https://github.com/mlflow/mlflow) for lineage tracking
+
+## Requirements
```

Comment:
should we mention GPU drivers?

Reply:
The drivers specifically is a weird grey area. While the docs will help people on setting this up, even in a future iteration we can't handle the driver installation in the DSS (even if we had a non-strict snap).

There's also distribution issues with the drivers so the drivers (NVIDIA) can't be part of the DSS, but rather the DSS can help set those up.

I kept tho the "Seamless GPU utilisation" since this includes the work we do on the K8s side to enable the GPU

Reply:
I'm ok with this for now, as you mentioned we will have the docs on setup so that will complete the story